src/leapr.hpp

```diff
@@ -120,43 +132,74 @@ auto finalLEAPR( nlohmann::json jsonInput ){
     double emax = 5.0;
     auto coherentElasticOut = coherentElastic( iel, npr, bragg, emax );
     nedge = std::get<1>(coherentElasticOut)* 0.5;
-    bragg.resize(std::get<0>(coherentElasticOut)); 
+    bragg.resize(std::get<0>(coherentElasticOut));
     braggOutput = bragg;
   }
-  else { 
-    braggOutput = false; 
+  else {
+    braggOutput = false;
     bragg.resize(0);
   }
 
   //---------------------------- Write Output --------------------------------
 
-  int za = jsonInput["za"];
-  std::vector<double> awrVec {awr,aws};
-  if (numSecondaryScatterers == 0){ awrVec.resize(1); }
+  njoy::ENDFtk::file::Type<7> MF7 = endout( jsonInput, sab_AllTemps, 
```

Comment:
This would work best in a separate function, maybe even call it `writeOutput`. That helps keep the call operator cleaner and easier to read.